src/environments/environment.development.ts

```diff
@@ -1,5 +1,5 @@
 export const environment = {
   production: false,
   abuseIpUrl: 'https://abuse-ip-proxy-ztbykxad.vercel.app/api/v2',
-  abuseIpApiKey: '',
-};
+  abuseIpApiKey: '5780de24e44ccd5416a7e8ec55b1df235c88ee470e97e7f2024827669e9f6b8cdcd44a03a8eef901',
```

Comment:
Pode remover esta key? Não devemos commitá-la.

```suggestion
  abuseIpApiKey: '',
```